src/bot/webserver.py

```diff
@@ -0,0 +1,31 @@
+from src.bot.app import bot
+from src.config import bot_settings
+from fastapi import FastAPI, Response, status
+from pydantic import BaseModel
+from src.dependendies.auth import VerifiedDep
+from src.schemas.auth import VerificationSource
+from src.exceptions import ForbiddenException
+import datetime
+
+
+class BookingInfo(BaseModel):
```

Comment:
Maybe it's better to mount this webhook to bot's process? Like it's commonly done for bot&telegram webhook but just do not invocate bot.set_webhook method.